projects/sandbox/src/app/_mocked/mocked-record-report.ts

```diff
@@ -57,11 +62,11 @@ export const mockMediaResources: Array<MediaDataItem> = [
 
 export const mockRecordReport: RecordReport = {
   recordTierCalculationSummary: {
-    europeanaRecordId: 'europeanaRecordId',
+    europeanaRecordId: mockProcessedRecordData.europeanaRecordId,
     providerRecordId: 'providerRecordId',
     contentTier: '4',
     metadataTier: 'C',
-    portalRecordLink: 'https://example.com'
+    portalRecordLink: mockProcessedRecordData.portalRecordLink
```

Comment:
I've modified the service so that it polls the dataset endpoint until its status is complete before fetching the data from the tier-calculation endpoint.

The panel will remain open while loading and the opener icon will temporarily be a spinner.  The links within the panel are shown disabled (with spinner icons next to each) until their urls become available.

Reply:
ok